libtransmission/session.h

```diff
@@ -1072,6 +1076,9 @@ struct tr_session
     // depends-on: session_thread_, settings_.bind_address_ipv6, local_peer_port_
     std::optional<BoundSocket> bound_ipv6_;
 
+    // depends-on: timer_maker_
+    std::unique_ptr<tr_global_ip_cache> global_ip_cache_ = std::make_unique<tr_global_ip_cache>(this);
```

Comment:
This actually also depends on `web_`, need to move it down.

Reply:
`udp_core_` now depends on the new field `global_ip_cache_` as well.

https://github.com/transmission/transmission/blob/f5c35c7855d57339390ed5011cdfa603be88eb9c/libtransmission/session.h#L1083-L1085